recipe-server/normandy/settings.py

```diff
@@ -333,6 +333,7 @@ def RAVEN_CONFIG(self):
     AWS_ACCESS_KEY_ID = values.Value()
     AWS_SECRET_ACCESS_KEY = values.Value()
     AWS_STORAGE_BUCKET_NAME = values.Value()
+    AWS_QUERYSTRING_AUTH = values.BooleanValue(False)
```

Comment:
I think this should probably be a part of `Core`, where settings we don't expect to change live. Either that, or this new configurable setting needs documentation.